core-bundle/contao/languages/en/default.xlf

```diff
@@ -2270,6 +2270,21 @@
       <trans-unit id="MSC.skipToContent">
         <source>Skip to content</source>
       </trans-unit>
+      <trans-unit id="MSC.searchIndexer.0">
+        <source>Search indexer</source>
+      </trans-unit>
+      <trans-unit id="MSC.searchIndexer.1">
+        <source>Include or exclude this page from the search index (Contao Crawler).</source>
+      </trans-unit>
+      <trans-unit id="MSC.searchIndexerBlankOption">
+        <source>Use robots tag</source>
+      </trans-unit>
```

Comment:
I think with my above suggestion we can remove the `blankOptionLabel` also from `tl_page`, since it will be clear that if you set nothing, the robots meta tag will be in effect.